alacritty/Cargo.toml

```diff
@@ -11,16 +11,16 @@ rust-version = "1.60.0"
 
 [dependencies.alacritty_terminal]
 path = "../alacritty_terminal"
-version = "0.17.1-dev"
+version = "0.17.1-rc1"
```

Comment:
We did bump rust version, so it should be `0.18.0-dev`.

Reply:
Bumped it since the changes in alacritty_terminal were nontrivial.